casa/IO/AipsIO.h

```diff
@@ -400,6 +403,10 @@ class AipsIO
     uInt getend();
 
 private:
+    // Copy constructor and assignment are not allowed.
+    AipsIO (const AipsIO&);
+    AipsIO& operator= (const AipsIO&);
```

Comment:
C++11 allows more explicit deletion:
```suggestion
    AipsIO (const AipsIO&) = delete;
    AipsIO& operator= (const AipsIO&) = delete;
```

Reply:
I agree this would be a nice change. However, there are many more such constructions in casacore and I prefer to do them all in one PR.

Reply:
When making such changes it might be wise to look at a tool like clang-tidy which can simply do these mechanical changes automatically. Much faster, less error prone.